openmaptiles/imposm.py

```diff
@@ -55,8 +56,12 @@ def create_imposm3_mapping(tileset_filename):
                 definition.pop('_resolve_wikidata', None)
                 tables[table_name] = definition
             for tag_name, definition in mapping.get('tags', {}).items():
+                if tag_name == 'include':
+                    tags[tag_name] |= set(definition)
                 tags[tag_name] = definition
```

Comment:
you are overriding the above right after adding to it - should be an `else` there.  Also, you may want to add a type check (e.g. make sure `include` is a list of strings). 

In general, I think in that for loop we should be explicit - only allow the `include`, but prohibit everything else (raise an error)

Poke me on Slack, or I can update your PR in a bit

Reply:
Changed